### PR TITLE
Update SetPedCombatAttributes info

### PR DESCRIPTION
**Pull request by alberttheprince**
Added all the most recent information and fixed the link to the source.

I have additionally documented some info from online, the docs, script hook documentation, etc, but I didn't know if it was appropriate to add this.

https://gist.github.com/alberttheprince/bd083ce03ceb187e9f9ecc7de9aa0a13

### CODE REVIEW

File: PED/SetPedCombatAttributes.md [hunk 1]
```diff
@@ -22,96 +22,96 @@ enum CombatAttributes
...
+	BF_PreferNonAircraftTargets = 87,
+	BF_PreferKnownTargetsWhenCombatClosestTarget = 88,
+	BF_ForceCheckAttackAngleForMountedGuns = 89,
+	BF_BlockFireForVehiclePassengerMountedGuns = 90,
+	MAX_COMBAT_FLAGS = 91,
 };
 ```
 
-The source for this enum is [here](https://alexguirre.github.io/rage-parser-dumps/dump.html?game=gta5&build=2944#_0x0E8E7201).
+The source for this enum is [here](https://alexguirre.github.io/rage-parser-dumps/dump.html?build=3095&game=gta5#CCombatData__BehaviourFlags).
```
Comment (PsychoShock) [inline PED/SetPedCombatAttributes.md:114]: Since the enum is valid, I don't think having a external doc "confirming" is necessary.

**Review by PsychoShock — APPROVED** [review]
Hello,

This is super nice PR to complete this native! Good job. Please note I am giving my personnal feedbacks here.

Thanks :)

**Review by alberttheprince — COMMENTED** [review]
Removed link

**Review by 4mmonium — APPROVED** [review]
Hey, your enum was OK, but I provided the official one, because why not 🙂
Thanks!

**Pull request merged**